### DISCUSSION TITLE
Clarification on sleeping bodies concept

### DISCUSSION THREAD

**Discussion by MrOnlineCoder** [jrouwe/JoltPhysics#974] [Q&A] [answered]
Hi again,

I've seen multiple times concept of sleeping bodies mentioned in the docs and also a bunch of methods/enums related to it, however I can't find a good explanation for it. Can you please help me figure out a few points about it? 

1) How does the state of body being awake or alseep affects the simulation? Can sleeping bodies still be collided with?

2) Should I in anyway manually manage sleeping state of bodies, or Jolt will do all switches and calculations by itself? So far I understand the only place where it happens is in **Solve Position Constraints, Update Bodies Broad Phase**

> When objects move too little the body will be put to sleep.

3) What example can be where manually putting body into sleep (deactivating) can become handy?

Thanks in advance.

**Comment by jrouwe** [comment, marked as answer]
I added a section here: https://jrouwe.github.io/JoltPhysics/index.html#sleeping-bodies

**Reply by MrOnlineCoder** [reply]
Your speed of adding documentation is spectacular. Thanks!
